fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -0,0 +1,86 @@
+module Fastlane
+  module Actions
+    class UploadSymbolsToCrashlyticsAction < Action
+      def self.run(params)
+        params[:binary_path] ||= (Dir["/Applications/Fabric.app/**/upload-symbols"] + Dir["./Pods/**/upload-symbols"]).last
+        UI.user_error!("Please provide a path to the binary using binary_path:") unless params[:binary_path]
+
+        unless params[:api_token].to_s.length > 0
+          Dir["./**/Info.plist"].each do |current|
+            result = Actions::GetInfoPlistValueAction.run(path: current, key: "Fabric")
+            next unless result
+            params[:api_token] ||= result["APIKey"]
+          end
+        end
+        UI.user_error!("Please provide an api_token using api_token:") unless params[:api_token]
+
+        command = []
+        command << File.expand_path(params[:binary_path])
+        command << "-a #{params[:api_token]}"
+        command << "-p ios"
+        command << File.expand_path(params[:dsym_path])
+
+        return Actions.sh command.join(" ")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Upload dSYM symbolication files to Crashlytics"
+      end
+
+      def self.details
+        [
+          "This action allows you to upload additional symbolication files to Crashlytics.",
```

Comment:
hmm, are you also using upload-symbols to do that, and will it upload all dSYMs for a build (like if they have many subprojects that build/link frameworks, etc)?